### DISCUSSION TITLE
Package's development status?

### DISCUSSION THREAD

**Discussion by sfirke** [markfairbanks/tidypolars#240] [General]
Potential new user here, I am a tidyverse superuser who switched jobs and now uses Python instead of R.  This looks awesome.  I see the last release was October 2022 and the last merged PR was 5 months ago and wondered: does the low activity level indicate that the project is simply complete and stable, or that it's no longer being actively developed?  I'm asking in good faith as someone whose public repos fall into both categories 😁

**Comment by markfairbanks** [comment]
Currently it falls under "no longer being actively developed" and wouldn't work very well if you used it. `polars` was making breaking changes every week at one point so it was hard to keep up. Now that `polars` v1.0.0 was released `tidypolars` could probably be revamped relatively quickly.

The biggest issue moving forward comes down to the question - should it be built to work directly on polars DataFrames or should it keep the Tibble subclass? The polars DataFrame route would allow it to work better with other packages but would also require a bit more typing. This is the proposed move to a polars DataFrame: https://github.com/markfairbanks/tidypolars/issues/238. The Tibble subclass is what is found in the README.

**Comment by markfairbanks** [comment]
FYI `tidypolars` has now been updated and is compatible with polars >=1.0.0
